database/ve_db_dev_data.sql

```diff
@@ -5,48 +5,58 @@
 -- This is intended to be temporary until we get full flow working with
 -- real data.
 -- ----------------------------------------------------------------------------
+INSERT INTO system_platform (uuid, rh_account, display_name, s3_url, vmaas_json, json_checksum) VALUES
+('PLAT-ID00-0000-0000', '00000000', 'deathstar.bogus.com', 'https://localhost/s3/111111', '{ package_list: { a }}', '11111111'),
```

Comment:
Looks like it's not a valid JSON. Strings should be wrapped in double quotes. Let's try to insert a real one. Maybe checksum should be a real as well.

Reply:
I've always used single quotes in sql statements.  I don't recall why, but it doesn't make a difference if its single or double quoted as far as I know.

The purpose of this initial dev/sample data was to support work on the manager.  There should be no reason for manager to care about s3_url, vmaas_json, or the checksum, so I put in short, bogus data just to have something inserted into these fields.  My expectation is that we would soon have listener component writing to these fields, followed by evaluator writing to system_vulnerabilities table.  If that is the path, it makes no sense to spend the time to put "real" data about "real" systems into the sample/dev data, plus the size of the actual data going into these fields would make the sample/dev data sql to insert it much more difficult to read.

If evaluator is ready to start reading and using these values, we can make the change.  However, without getting real data from Upload Service (providing actual s3_url's) the only thing evaluator can do is send a package list to vmaas about a bogus system.

Reply:
I think I see what you mean about double quotes.  You're suggesting the "a" in the package list be surrounded by double quotes.  Since its not a real vmaas api POSTable json, it should matter what's in the db.